tests/playwright-tests/admin/pages/CreateNewReleasePage.ts

```diff
@@ -0,0 +1,31 @@
+import { Locator, Page } from '@playwright/test';
+
+export default class CreateNewReleasePage {
+  readonly page: Page;
+  readonly academicType: Locator;
+  readonly academicYear: Locator;
+  readonly releaseType: Locator;
+  readonly createNewReleaseButton: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    // Locators
+    this.academicType = page.locator('//label[contains(text(),"Type")]');
+    this.academicYear = page.locator(
+      'input[name="timePeriodCoverageStartYear"]',
+    );
+    this.releaseType = page.locator(
+      'input[id="releaseSummaryForm-releaseType-NationalStatistics"]',
+    );
+    this.createNewReleaseButton = page.locator(
+      '//button[contains(text(),"Create new release")]',
+    );
+  }
+
+  async createNewRelease() {
```

Comment:
I guess that we'd change this to pass in "AYQ1" and "2022" as parameters when the next test came along that needed to use this page object?